frontend/src/components/dialogs/MemberDialog.vue

```diff
@@ -65,6 +65,15 @@ SPDX-License-Identifier: Apache-2.0
               ></v-text-field>
             </v-col>
           </v-row>
+          <v-alert
+            v-if="isUpdateDialog && orphaned"
+            :value="true"
+            type="info"
+            color="primary"
+            outlined
+          >
+            The service account will be created in case it does not exist
```

Comment:
This sounds like we do not know if the sa exits. maybe something like
`The service account does not exist anymore and will be re-created if you update the roles.`
or just `The service account will be re-created`